app/view/templates/admin.php

```diff
@@ -29,27 +29,27 @@
                     <p>Here you can set the home-page view for visitors.</p>
 
                     <div class="radio">
-                        <input type="radio" name="homepage" value="default" id="default" <?= Config::homepage() === 'default' ? 'checked' : '' ?>>
+                        <input type="radio" name="homepage" value="default" id="default" <?= Wcms\Config::homepage() === 'default' ? 'checked' : '' ?>>
```

Comment:
The same happens in the templates as they are not in the `Wcms` namespace either.